test/test_pf_against_pandapower.py

```diff
@@ -17,66 +12,89 @@
+    # use same index for everything
     net.res_bus.index = net.bus.name.values
     net.res_line.index = net.line.name.values
 
-    #compare bus angles
-    np.testing.assert_array_almost_equal(n.buses_t.v_ang.loc["now"]*180/np.pi,net.res_bus.va_degree)
-
-    #compare bus voltage magnitudes
-    np.testing.assert_array_almost_equal(n.buses_t.v_mag_pu.loc["now"],net.res_bus.vm_pu)
-
-    #compare bus active power (NB: pandapower uses load signs)
-    np.testing.assert_array_almost_equal(n.buses_t.p.loc["now"],-net.res_bus.p_mw)
-
-    #compare bus active power (NB: pandapower uses load signs)
-    np.testing.assert_array_almost_equal(n.buses_t.q.loc["now"],-net.res_bus.q_mvar)
-
-    #compare branch flows
-    np.testing.assert_array_almost_equal(n.lines_t.p0.loc["now"],net.res_line.p_from_mw)
-    np.testing.assert_array_almost_equal(n.lines_t.p1.loc["now"],net.res_line.p_to_mw)
-    np.testing.assert_array_almost_equal(n.lines_t.q0.loc["now"],net.res_line.q_from_mvar)
-    np.testing.assert_array_almost_equal(n.lines_t.q1.loc["now"],net.res_line.q_to_mvar)
-
-    np.testing.assert_array_almost_equal(n.transformers_t.p0.loc["now"],net.res_trafo.p_hv_mw)
-    np.testing.assert_array_almost_equal(n.transformers_t.p1.loc["now"],net.res_trafo.p_lv_mw)
-    np.testing.assert_array_almost_equal(n.transformers_t.q0.loc["now"],net.res_trafo.q_hv_mvar)
-    np.testing.assert_array_almost_equal(n.transformers_t.q1.loc["now"],net.res_trafo.q_lv_mvar)
+    # compare bus angles
+    np.testing.assert_array_almost_equal(
+        n.buses_t.v_ang.loc["now"] * 180 / np.pi, net.res_bus.va_degree
+    )
+
+    # compare bus voltage magnitudes
+    np.testing.assert_array_almost_equal(
+        n.buses_t.v_mag_pu.loc["now"], net.res_bus.vm_pu
+    )
+
+    # compare bus active power (NB: pandapower uses load signs)
+    np.testing.assert_array_almost_equal(n.buses_t.p.loc["now"], -net.res_bus.p_mw)
+
+    # compare bus active power (NB: pandapower uses load signs)
+    np.testing.assert_array_almost_equal(n.buses_t.q.loc["now"], -net.res_bus.q_mvar)
+
+    # compare branch flows
+    np.testing.assert_array_almost_equal(
+        n.lines_t.p0.loc["now"], net.res_line.p_from_mw
+    )
+    np.testing.assert_array_almost_equal(n.lines_t.p1.loc["now"], net.res_line.p_to_mw)
+    np.testing.assert_array_almost_equal(
+        n.lines_t.q0.loc["now"], net.res_line.q_from_mvar
+    )
+    np.testing.assert_array_almost_equal(
+        n.lines_t.q1.loc["now"], net.res_line.q_to_mvar
+    )
+
+    np.testing.assert_array_almost_equal(
+        n.transformers_t.p0.loc["now"], net.res_trafo.p_hv_mw
+    )
+    np.testing.assert_array_almost_equal(
+        n.transformers_t.p1.loc["now"], net.res_trafo.p_lv_mw
+    )
+    np.testing.assert_array_almost_equal(
+        n.transformers_t.q0.loc["now"], net.res_trafo.q_hv_mvar
+    )
+    np.testing.assert_array_almost_equal(
+        n.transformers_t.q1.loc["now"], net.res_trafo.q_lv_mvar
+    )
```

Comment:
I guess this is only line breaks or?

Reply:
Yes, I have mapplied black coding style to this script as I was unhappy with the formatting of lines 41ff.